qbittorrent-wireguard/.env-dist

```diff
@@ -1,5 +1,5 @@
 QBITTORRENT_TRAEFIK_HOST=transmission.example.com
-QBITTORRENT_VERSION=4.5.4
+QBITTORRENT_VERSION=amd64-4.6.4
```

Comment:
This probably wont work on arm then.

Reply:
Fixed in [084cf93](https://github.com/EnigmaCurry/d.rymcg.tech/pull/288/commits/084cf937c661476368bccb4111d0d11cff38418b)